package.json

```diff
@@ -6,13 +6,19 @@
     "name": "Kevin Lindsey",
     "email": "kevin@kevlindev.com"
   },
+  "contributors": [
```

Comment:
I've been remiss here. We should add "Robert Benko" as well

Reply:
Ok, done.

Reply:
Thanks for that. I would likely forget to do that later :)

Reply:
Fully understood. And FWIW, I mostly just added myself, to avoid seeing the package.json linter complaining whenever I had the file open. :)